### PR TITLE
Updates for v0.4.0

### PR DESCRIPTION
**Pull request by JonathanAnderson**
I cleaned this up to remove the warnings i was getting when using this package with branch `v0.4.0`

### CODE REVIEW

**Comment by mlubin** [comment]
@JonathanAnderson, thanks for this (ref #59). The code outside of ``Mosek.jl`` and ``MosekSolverInterface.jl`` is automatically generated. If you rebase out your changes there we can merge this and let @ulfworsoe handle the rest.

**Comment by JonathanAnderson** [comment]
I've amended the commit so it only changes `Mosek.jl` and `MosekSolverInterface.jl`

**Comment by JonathanAnderson** [comment]
It looks like some of the code in `msk_functions.jl` was already generated with `UInt8` but other parts generated with `Uint8`.

With the changes above (and manual modification of `msk_functions.jl` and `msk_callback.jl`), I can run a quadratic optimization without any deprecation warnings.

File: deps/liftedfrombindeps.jl [hunk 1]
```diff
@@ -2,7 +2,7 @@
 # script sometimes cannot find BinDeps.
 # @windows_only unpack_cmd has been modified to use zip in addition to 7z.
 
-function splittarpath(path) 
+function splittarpath(path)
```
Comment (mlubin) [inline deps/liftedfrombindeps.jl]: Also need ``using Compat`` in this file

**Comment by mlubin** [comment]
Looks good, could you squash the commits into one?

**Comment by JonathanAnderson** [comment]
Of course



—Jon

**Comment by mlubin** [comment]
Thanks!

**Pull request merged**